src/components/shared/NewDeployNotAvailable.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro'
+import { Button } from 'antd'
+
+import { useHistory } from 'react-router-dom'
+
+import { BigNumber } from '@ethersproject/bignumber'
+
+import { layouts } from 'constants/styles/layouts'
+import { padding } from 'constants/styles/padding'
+
+export default function NewDeployNotAvailable({
+  name,
+}: {
+  name?: string | BigNumber
```

Comment:
Yeah this is good, but maybe now that we're not passing `projectMetadata.name` from `V2Dashboard`,  we can change this `name` prop to `handleOrId` ?

Reply:
Done 💥